src/coprocessor/xeval/builtin_math.rs

```diff
@@ -111,57 +123,162 @@ mod test {
         };
     }
 
-    test_eval!(test_abs_int,
-               vec![(build_expr_with_sig(vec![Datum::I64(-1)],
-                                         ExprType::ScalarFunc,
-                                         ScalarFuncSig::AbsInt),
-                     Datum::I64(1)),
-                    (build_expr_with_sig(vec![Datum::I64(1)],
-                                         ExprType::ScalarFunc,
-                                         ScalarFuncSig::AbsInt),
-                     Datum::I64(1)),
-                    (build_expr_with_sig(vec![Datum::U64(1)],
-                                         ExprType::ScalarFunc,
-                                         ScalarFuncSig::AbsInt),
-                     Datum::U64(1)),
-                    (build_expr_with_sig(vec![Datum::Null],
-                                         ExprType::ScalarFunc,
-                                         ScalarFuncSig::AbsInt),
-                     Datum::Null)]);
+    test_eval!(
+        test_abs_int,
```

Comment:
One PR one thing, please do not update or reformat the block not related to your PR. Thanks anyway.